packages/astro/src/@types/astro.ts

```diff
@@ -2901,13 +2902,47 @@ export interface ClientDirectiveConfig {
 	entrypoint: string;
 }
 
-export interface DevToolbarApp {
+type DevToolbarAppMeta = {
 	id: string;
 	name: string;
 	icon?: Icon;
+};
+
+export type DevToolbarApp = {
```

Comment:
The types are quite messy around these parts right now because we support both patterns, once we remove the old pattern, there'll only be one type again